pkgs/applications/science/physics/xnec2c/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchurl
+, autoreconfHook
+, pkg-config
+, which
+, gtk3
+, openblas
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xnec2c";
+  version = "4.4.12";
+
+  src = fetchurl {
+    url = "https://www.xnec2c.org/releases/${pname}-v${version}.tar.gz";
+    hash = "sha256-6Yrx6LkJjfnMA/kJUDWLhGzGopZeecARSrcR++UScsU=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config which ];
+  buildInputs = [ gtk3 openblas ];
+
+  meta = with lib; {
+    homepage = "https://www.xnec2c.org/";
+    description = "Graphical antenna simulation";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ mvs ];
+    platforms = platforms.unix;
```

I can not find a hint that is supposed to run on darwin.
```suggestion
    platforms = platforms.linux;
```

---

The `INSTALL` file specifically refers to creating fat binaries on MacOS X.

The latest `configure.ac` in the development branch contains code to deal with some of its peculiarities. I assume that compilation on Darwin is going to be fixed with the next release. At least the dependency on `<link.h>` has been removed.